packages/webapp/src/components/Form/Unit/index.jsx

```diff
@@ -381,6 +382,7 @@ const Unit = ({
               styles.verticleDivider,
               showError && styles.inputError,
               disabled && styles.none,
+              hideLineUnitSeparator && styles.none,
```

Comment:
@wasswarichard 

Regarding the line separator on the units. I 100% changed that functionality in the [Fixing issues where units were not being displayed in the disabled units by kikemarquez8 · Pull Request #2323 · LiteFarmOrg/LiteFarm](https://github.com/LiteFarmOrg/LiteFarm/pull/2323). But I also don’t see it spec'ed in [Unit displays](https://lite-farm.atlassian.net/wiki/spaces/LITEFARM/pages/658309153/Unit+displays). I have asked @kcussen if I can revert this outside of this story if that is preferable.

Reply:
@Duncan-Brain @anant00729 please review the changes

Reply:
@wasswarichard 

Based off the discussion today please remove this line all references to this 'hideLineUnitSeperator' . Since I messed with the units in the other PR we will fix this elsewhere. Including in WaterUseCalculator